lib/matplotlib/cbook/__init__.py

```diff
@@ -2027,3 +2066,4 @@ def _setattr_cm(obj, **kwargs):
                 delattr(obj, attr)
             else:
                 setattr(obj, attr, orig)
+
```

Comment:
Looks like a PEP8 check is not liking the blank line here? I thought we were supposed to have a blank line at the end of files?

Reply:
A new line, not a full blank line.

Reply:
Ah, I couldn't tell that with the github diffs. I like how git diff shows you those trailing blank lines.

Reply:
Bad rebase I guess - will fix